types/serverless/index.d.ts

```diff
@@ -77,7 +78,7 @@ declare class Serverless {
     getVersion(): string;
 
     cli: {
-        log(message: string): null;
+        log(message: string, entity?: string, options?: object): null;
```

Comment:
```suggestion
        log(message: string, entity?: string, options?: { color?: string }): null;
```

We know that `color` is at least one of the supported options - I've not actually found any other options glancing over the codebase.